app/helpers/audits_helper.rb

```diff
@@ -3,7 +3,7 @@ module AuditsHelper
                     Location Organization Domain Subnet SmartProxy AuthSource Image Role Usergroup Bookmark ConfigGroup Ptable)
 
   # lookup the Model representing the numerical id and return its label
-  def id_to_label(name, change, truncate = true)
+  def id_to_label(name:, change:, audit: @audit, truncate: true)
```

Comment:
you can mix both regular args with kwargs, so that the default call doesn't change at all, e.g.:
```ruby
def id_to_label(name, change, audit: @audit, truncate: true)
...
```